src/internal/components/chart-popover/__integ__/chart-popover.test.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+
+import createWrapper from '../../../../../lib/components/test-utils/selectors';
+
+describe.each([false, true])('iframe=%s', iframe => {
+  test(
+    'can be unpinned by clicking outside',
+    useBrowser(async browser => {
+      const page = new BasePageObject(browser);
+      await browser.url(`#/light/charts.test?iframe=${iframe}`);
+      await page.runInsideIframe('#content-iframe', iframe, async () => {
+        const chart = createWrapper().findLineChart();
+        const popover = chart.findDetailPopover();
+        const popoverDismiss = popover.findDismissButton();
+
+        // Pins popover on the first point.
+        await page.click('h2');
+        await page.keys(['Tab', 'ArrowRight', 'Enter']);
```

Comment:
@gethinwebster I changed popover activation from pointer to keyboard because the former approach did not work with React 16 test pages: the page.hoverElement() did not help with offsetting the click. I did test the feature manually on both React versions.